capabilities/devops.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: capability
+
+asset-name: devops
+
+description: "DevOps is the ability to do multiple operations simultaneously, improving
+efficiency and speed of deployment. See how we do it."
+
+tags: [DevOps]
+
+title: DevOps
+
+subtitle: "It’s developers and operations working together to safely (but quickly) get your code out to your servers."
+
+text: '<p>We do DevOps for all our clients. </p>
+<p>Let’s say you have a database, or two, or 300. There’s your security component, (link
+to security page), the authentication and authorization system. There’s a session server
+to store security information for the user. There’s an API server that accesses the
+database and sends the information around. Those are ready to serve for caching. You
+might have a bunch of different systems, a bunch of different servers (or containers) of
+data. All those things need to come up in a very specific order. DevOps is the process
+which ensures that happens.</p>
+<p>Or maybe you’re just focused on one piece. Say one of five servers need to be updated.
+How do you do that in a way so the other four don’t fail?</p>
+<p>Operations sets up the infrastructure. We write your code. Then we test the code. If
+any of those tests fails, we go back and fix it. After that, the code moves out to
+servers.</p>
+<p>Developers do certain things to make the operations team feel good about what they’re
+doing. The operations team does certain things to make the developers feel confident in
+their work.</p>
+<p>The intersection of Agile development and people is DevOps.</p>'
```

Comment:
This article doesn't explain the topic accurately enough. We need to discuss this further.

Reply:
Sounds good

Reply:
Fixed